chihuahua/assetlist.json

```diff
@@ -57,5 +57,31 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chihuahua/images/puppyhuahua_logo.png"
       }
     }
-  ]
-}
\ No newline at end of file
+  ],
+  
+  {
+      "description": "has a hat",
+      "denom_units": [
+        {
+          "denom": "factory/chihuahua1x4q2vkrz4dfgd9hcw0p5m2f2nuv2uqmt9xr8k2/achihuahuawifhat",
+          "exponent": 0
+        },
+        {
+          "denom": "achihuahuawifhat",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/chihuahua1x4q2vkrz4dfgd9hcw0p5m2f2nuv2uqmt9xr8k2/achihuahuawifhat",
+      "name": "Chihuahuawifhat",
+      "display": "chihuahuawifhat",
```

Comment:
there is no unit of this name defined. Is this supposed to be 'achihuahuawifhat'?

Reply:
Yes sorry,

Reply:
I think the name was better before. And the `display` value is still invalid.